components/gitpod-db/src/typeorm/workspace-cluster-db-impl.ts

```diff
@@ -54,6 +54,8 @@ export class WorkspaceClusterDBImpl implements WorkspaceClusterDB {
             state: "available",
             govern: false,
             admissionConstraints: [],
+            availableWorkspaceClasses: [],
+            preferredWorkspaceClass: "",
```

Comment:
I don't think the prototype declaration and the select below are necessary. 
@geropl do you remember the reason you did this 3years ago?

Reply:
Without this prototype the available workspace classes are never returned. Cost me a day to find - and only with Gero's help did we fix it.

Reply:
I removed the whole prototype and the select and the tests were green. I don't understand why we would have this select thing here as we usually don't need that. That's why I pinged Gero.

Reply:
This got introduced to not select the `tls` field from the database, which is quite large, and had performances impacts while polling. :slightly_smiling_face:
The type is wrong, it should be: `Required<WorkspaceClusterWOTLS>` I think, which would have fixed this problem.